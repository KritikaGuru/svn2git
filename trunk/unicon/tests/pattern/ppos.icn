procedure main()
   out := &output
   s := "abcda"
   if s ?? PPos(1) && "b" then
      write("Match succeeded")
   else
      write("Match failed")
   s ?? PLen(3) -> out && PRpos(0) 
   s ?? PPos(4) && PLen(1) -> out
   if s ?? PPos(1) && "abcd" && PRpos(0) then
      write("Match succeeded")
   else
      write("Match failed")   out := &output
   s := "abcda"
   s ?? PLen(3) -> out
   s ?? PLen(2) -> out && "a"
end
