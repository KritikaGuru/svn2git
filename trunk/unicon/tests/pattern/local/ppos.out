-----
Match succeeded for: abcdef
def
d
Match failed for: abcdef
-----
Match failed for: bacdef
def
d
Match failed for: bacdef
-----
Match succeeded for: abcd
bcd
d
Match succeeded for: abcd
